artcommon/artcommonlib/assembly.py

```diff
@@ -381,3 +381,7 @@ def assembly_metadata_config(releases_config: Model, assembly: typing.Optional[s
             config_dict = _merger(component_entry.metadata.primitive(), config_dict)
 
     return Model(dict_to_model=config_dict)
+
+
+def contains_embargoed_build(releases_config: Model, assembly: str):
```

Comment:
I'm not sure what the utility of having it defined in assembly is .. or what we would be preventing.